lldb/include/lldb/Target/Process.h

```diff
@@ -1129,10 +1129,15 @@ class Process : public std::enable_shared_from_this<Process>,
   /// \see Thread:Resume()
   /// \see Thread:Step()
   /// \see Thread:Suspend()
-  virtual Status DoResume() {
+  virtual Status DoResume(lldb::RunDirection direction) {
```

Comment:
I would rather have a new `DoResumeReverse()` instead of changing this virtual API. Many modified files in this patch are a result of the process plugins having to add support for not supporting reverse resumes.

Reply:
The decision was to use a direction flag.